File: OrnaTrack/MyUserControls/OneDayControl.vb
Public Class OneDayControl
    Public _kitty As Kitty

    Private Sub Panel1_DoubleClick(sender As Object, e As EventArgs) Handles Panel1.DoubleClick
        Cursor = Cursors.AppStarting
        Using Fm As New PreviewKitty With {
            ._kitty = _kitty
        }
            Fm.ShowDialog()
        End Using
        Cursor = Cursors.Default
    End Sub

End Class

File: OrnaTrack/MyUserControls/OneDayControl.cs
using System;
using System.Windows.Forms;

namespace FirstApp
{
    public partial class OneDayControl
    {
        public Kitty _kitty;

        public OneDayControl()
        {
            InitializeComponent();
        }

        private void Panel1_DoubleClick(object sender, EventArgs e)
        {
            Cursor = Cursors.AppStarting;
            using (var Fm = new PreviewKitty() { _kitty = _kitty })
            {
                Fm.ShowDialog();
            }
            Cursor = Cursors.Default;
        }

    }
}